modules/file/file_test.go

```diff
@@ -102,6 +102,47 @@ func TestContentSearch(t *testing.T) {
 	}
 }
 
+func TestDecompressedContentSearch(t *testing.T) {
+	for _, tp := range TESTDATA {
+		var (
+			r run
+			s search
+		)
+		var expectedfiles = []string{""}
+		if tp.name != "testfile5" {
```

Comment:
I would prefer to do the opposite here, and have a special case that resets `expectedfiles` to an empty array if `tp.name == "testfile5"`.
A comment explaining that testfile5 is a corrupt gzip that should fail to decompress would be nice too.